frontend/integration-tests/tests/kubevirt/utils/consts.ts

```diff
@@ -0,0 +1,75 @@
+
+// TIMEOUTS
+const SEC = 1000;
+export const CLONE_VM_TIMEOUT = 300 * SEC;
```

Comment:
I would put `SECONDS` in each of these names to make it obvious what the unit is where it's used.

```suggestion
export const CLONE_VM_TIMEOUT_SECONDS = 300 * SEC;
```